src/commons/api/history.rs

```diff
@@ -428,6 +428,12 @@ pub enum StorableCaCommand {
     ChildRemove {
         child: ChildHandle,
     },
+    ChildSuspendInactive {
+        child: ChildHandle,
+    },
+    ChildUnsuspend {
```

Comment:
I think I like Unsuspend as it is clearly the opposite of suspend, but I can't help thinking of "resume" as another term that could be used... perhaps unsuspend is best...